arguments/SPIR-V.argdown

```diff
@@ -60,6 +61,11 @@ functionality present in core SPIR-V, like the instructions for ray tracing.
 as well as huge shader code bases. Adding support for another one is something nobody is excited about.
   +> [SPIR-V]
 
+<Limit discord and fragmentation between browsers>:
+  + Less implementation and debate effort means less ways to fragment the ecosystem.
+  + WebGPU will be less web centric and in a better position to be used in native apps.
```

Comment:
I totally get that argument, but we can't raise this in the context of W3C designing the API.
The goal is the Web, everything else is based on initiative of individual participants, like us and Google.
Saying that, I think the line will need to be removed.

Reply:
"Beyond the current web limitation and competing directly with native apps"? :)